builtin/logical/transit/policy.go

```diff
@@ -25,6 +30,11 @@ const (
 	ErrTooOld = "ciphertext version is disallowed by policy (too old)"
 )
 
+const (
```

Comment:
`iota` are per-const-block. Technically it could be done because it starts at the first use of `iota` in a const block but I feel a little icky about it. Maybe I just need to get over that :-)